store/queries/candidate_posts.sql

```diff
@@ -66,28 +82,34 @@ FROM
 INNER JOIN candidate_actors AS ca ON cp.actor_did = ca.did
 INNER JOIN post_scores AS ph
     ON
-        ph.uri = cp.uri AND ph.alg = sqlc.arg(alg)
+        cp.uri = ph.uri AND ph.alg = sqlc.arg(alg)
         AND ph.generation_seq = sqlc.arg(generation_seq)
 WHERE
     cp.is_hidden = false
     AND ca.status = 'approved'
+    -- Match at least one of the queried hashtags.
+    -- If unspecified, do not filter.
     AND (
         COALESCE(sqlc.arg(hashtags)::TEXT [], '{}') = '{}'
         OR sqlc.arg(hashtags)::TEXT [] && cp.hashtags
     )
+    -- Match has_media status. If unspecified, do not filter.
     AND (
         sqlc.narg(has_media)::BOOLEAN IS NULL
-        OR COALESCE(cp.has_media, false) = sqlc.narg(has_media)
+        OR COALESCE(cp.has_media, false) = sqlc.arg(has_media)
```

Comment:
Nit: It feels a little odd to use narg on the previous line and then arg here (although I don't think this affects the actual SQL generated by `sqlc`) - was this something the linter directed us to do ?